New post: DriveBackupV2 (#40)
FalixNodes-Software/help-center: _posts/minecraft/plugins/2021-8-10-backup.md

---
layout: post
title:  "DriveBackup"
categories: Minecraft
tags: plugins
permalink: /minecraft/plugins/drivebackup/
---

<div class="install-plugin">
    <img style="border-radius: 7px;" src="https://media.forgecdn.net/avatars/272/319/637250850729446954.png">
    <p>DriveBackupV2</p>
    <a href="https://dev.bukkit.org/projects/drivebackupv2/">Download this Plugin</a>
</div>

### What is DriveBackupV2?

> Drivebackup is a plugin that automatically backs up your server to the cloud, or to your own device!

Reviewer: ```suggestion
DriveBackupV2 is a plugin that will automatically backup your server to the cloud, or to your own device!
```

## Setting up DriveBackupV2

1. Set up the server, you can use Spigot or any of its forks, we recommend using [Purpur](https://purpur.pl3x.net) or [Paper](https://papermc.io).
2. Open the file manager.
3. Find and click on the folder named "plugins".

> 4. Download the [DriveBackupV2](https://dev.bukkit.org/projects/drivebackupv2/) and upload it to the folder.

Reviewer: ```suggestion
4. Download the [DriveBackupV2](https://dev.bukkit.org/projects/drivebackupv2/) plugin and upload it to the folder.
```

5. Restart the server.
6. Go back to the "plugins" folder, you should see that a new folder was generated, it is usually named "DriveBackupV2".

> 7. Read the [docs](https://github.com/MaxMaeder/DriveBackupV2/wiki) for instructions on how to set it up.

Reviewer: ```suggestion
7. Read the [plugin wiki](https://github.com/MaxMaeder/DriveBackupV2/wiki) for instructions on how to set it up.
```

> 8. once you are done setting up everything, restart the server to load changes.

Reviewer: ```suggestion
8. Once setup is done, restart the server to load your changes.
```
